jobfunnel/tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+from unittest.mock import patch
+from ..config.parser import parse_config
+from ..tools.tools import config_factory
+import sys
+
+
+@pytest.fixture()
+def configure_options():
+    def setup(options):
+        """Assigns the options to argv(as if JobFunnel were called from the command line with those options)
+        and calls parse_config(). This fixture assumes that the test_parse module has been tested
+        and passes.
+        """
+        with patch.object(sys, 'argv', options):
+            config = parse_config()
+        return config
+
+    return setup
+
+
+@pytest.fixture()
+def job_listings():
+    def setup(attr_list):
+        job_format = {'status': 'new', 'title': 'Python Engineer', 'company': 'Python Corp', 'location': 'Waterloo, ON', 'date': '10 days ago', 'blurb': '', 'tags': '',
```

Comment:
lets wrap this at 100 chars.

Reply:
The huge hyperlink messed things up here. Has been fixed on 7b675c906fe448956b0a506a1e860c4d2fee1172
I did this through Atom, hopefully it didn't mess up this time :)